src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtil.java

```diff
@@ -384,8 +384,7 @@ public static boolean isNonVoidMethod(DetailAST methodDefAst) {
      * @return true if the parameter is a receiver.
      */
     public static boolean isReceiverParameter(DetailAST parameterDefAst) {
-        return parameterDefAst.getType() == TokenTypes.PARAMETER_DEF
-                && parameterDefAst.findFirstToken(TokenTypes.IDENT) == null;
+        return parameterDefAst.findFirstToken(TokenTypes.IDENT) == null;
```

Comment:
Right now this method doesn't not make any sense and should be inlined to each Check as `parameterDefAst.findFirstToken(TokenTypes.IDENT) == null` it is not util method anymore.

Reply:
done